### DISCUSSION TITLE
Issues installing

### DISCUSSION THREAD

**Discussion by shot-codes** [nix-community/kickstart-nix.nvim#27] [Q&A]
I belive I've manged to install on nixos with flake, but it doesn't seem like plugins are working. The color theme has changed though. Also, i just cloned my kickstart-nix.nvim repo into .config/nvim, is that the idea?

**Comment by mrcjkb** [comment]
Hey :wave:

you're probably running into this issue (see the readme's: [installation section](https://github.com/nix-community/kickstart-nix.nvim?tab=readme-ov-file#snowflake-nixos-with-flakes)):

> [!IMPORTANT]
>
> This flake uses `nixpkgs.wrapNeovimUnstable`, which has an
> unstable signature. If you set `nixpkgs.follows = "nixpkgs";`
> when importing this into your flake.nix, it may break.
> Especially if your nixpkgs input pins a different branch.

Also, see [this discussion](https://github.com/nix-community/kickstart-nix.nvim/discussions/16).

> Also, i just cloned my kickstart-nix.nvim repo into .config/nvim, is that the idea?

No, there's no need to clone it there. You can put it wherever you like.
I just have a separate repo for my neovim config that I add as an input into my nixos configs flake.
